beacon-chain/rpc/eth/shared/structs_request_and_responses.go

```diff
@@ -0,0 +1,7 @@
+package shared
+
+type StateForkResponse struct {
```

Comment:
- this type should be placed next to the handler
- the name should ideally reflect the function name: `GetStateForkResponse`